### PR TITLE
deleting only OFN related Docker containers and images

### PR DESCRIPTION
**Pull request by turinglabsorg**
Developers can host several different `images` and `containers` even not related to openfoodnetwork. This PR will fix any issue related to the use of `docker/nuke` so any developer can easily run this script without removing any other container/image.

### CODE REVIEW

**Review by luisramos0 — APPROVED** [review]

**Comment by andrewpbrett** [comment]
Thanks for this contribution - it looks like there's a change to a `.scss` file included as well. Was that intentional? It looks unrelated to the docker script.

**Comment by turinglabsorg** [comment]
Yes it'sintentional, I jumped into this project https://github.com/openfoodfoundation/openfoodnetwork/projects/27 and found this issue https://github.com/openfoodfoundation/openfoodnetwork/issues/5916. Changes in `.scss` code are related to that issue!

**Comment by andrewpbrett** [comment]
Got it, thanks @turinglabsorg. Would you mind splitting this into two separate PRs so we can review/test them independently?

**Comment by turinglabsorg** [comment]
Sure, i just removed useless commits here and created a new PR #6873 with the fix. Thank you for your suggestion, next PRs will be focused on a single fix / improvement.

**Review by andrewpbrett — APPROVED** [review]
Great, thank you! There are some other contributing tips in the [wiki](https://github.com/openfoodfoundation/openfoodnetwork/wiki/Making-a-great-pull-request). This one looks great and now that it has two approvals, I think it can be merged once the automated tests pass. Usually after two approvals it goes to our testing team but I don't think that's necessary here so I'm adding the pr-no-test label. That process is also on the [wiki](https://github.com/openfoodfoundation/openfoodnetwork/wiki/The-process-of-review%2C-test%2C-merge-and-deploy) if you want to take a look. 

Thanks again for the contribution!

**Pull request merged**
